src/react/workflow/details/Configuration.jsx

```diff
@@ -62,7 +54,7 @@ function Configuration({ wfSelected, replications, bucketList, locations, showEd
                         </T.BannerContainer>
                         <Button icon={<i className="fas fa-trash" />} text="Delete Workflow" variant='buttonDelete' onClick={handleOpenDeleteModal} />
```

Comment:
"Banner" and "deletion button" could be moved to `src/react/workflow/WorkflowEditor.jsx` and only be shown when createMode is set to `false`. That way we might avoid having to (re)define `showEditWorkflowNotification` in three deferent component, hence avoiding extra rendering.

Reply:
Code review is expected from @nicolas2bert .